tests/unit-tests/Filesystem/LoadsTranslationsTest.php

```diff
@@ -91,10 +91,6 @@ public function creates_namespace_if_not_global_override()
 
     protected function saveAndConfirmTenantTranslation()
     {
-        $original = include base_path('resources/lang/en/passwords.php');
-
-        $this->assertEquals($original['password'], trans('passwords.password', [], 'en'));
-
```

Comment:
Not sure if this will impact the package? I removed this check, because the `password` entry no longer exists in Laravel 6.

Reply:
Don't remove this, you just invalidated part of our tests. Pick any other translation key and match it up please.

Reply:
@luceos  Can you elaborate? Won't picking a random translation key break the test? What is the goal of this check -- perhaps by better understanding what is being tested for, I can implement a proper solution.

Would you like me to test any other of the items, like so:
```php
$this->assertEquals($original['reset'], trans('passwords.reset', [], 'en'));
```